Sources/AdsSDKSwift/ZestyBannerView.swift

```diff
@@ -90,6 +110,29 @@ public struct ZestyBannerView: View {
             }
         }
     }
+    
+    private func calculateScale() -> CGSize {
```

Comment:
Could we also use the `CGRect` classes here?

Reply:
For which part? I figured since scale factor takes a CGSize I would just work with that in the function

Reply:
Thought that CGSize values could sometimes be negative so far additional safety could use a CGRect to be able to standardize and ensure positive values. Probably not needed here.

Reply:
Ahhh I see, yeah for this I think it should be enough of a common sense thing for devs to not pass a negative value, but I can add some handling for it just to be safe.